src/main/java/pageobject/HelpersAction.java

```diff
@@ -0,0 +1,16 @@
+package pageobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class HelpersAction {
```

Comment:
No need in new class. Use your existing Helpers. Rename method clearAndInputWithAction()

Reply:
And Please move Helpers class to utils folder. It should not be in pageobject package. I missed it in previous review